### PR TITLE
Create Adding ERC-20 tokens to Lisk guide

### LINKED ISSUE #10: Create guide: How to add your Token to Lisk
### Description
Create a new guide for token issuers who already have an ERC-20 contract deployed on Ethereum and would like to submit their token for bridging between Ethereum and Lisk, and Lisk Mainnet to Lisk Bridge and other bridges. 

See [Base Token guide](https://docs.base.org/tokens/list) for additional information.

### Category

Building on Lisk

### PR DESCRIPTION
**Pull request by TalhaMaliktz**
### What was the problem?

This PR resolves #10

### CODE REVIEW

File: docs/about-lisk/network-info.md [hunk 1]
```diff
@@ -25,14 +25,14 @@ The Lisk L2 is available on the following network:
...
-| Currency Symbol     | ETH                                                                         |
-| Block Explorer      | https://sepolia-blockscout.lisk.com/                                        |
-| Lisk Testnet Bridge | https://sepolia-bridge.lisk.com                                             |
-| Lisk Testnet Faucet | https://app.optimism.io/faucet                                              |
+| **Network Name** | Lisk Sepolia Testnet                                                        |
+| **HTTP RPC Endpoint** | [https://rpc.sepolia-api.lisk.com](https://rpc.sepolia-api.lisk.com)        |
+| **WS RPC Endpoint** | [wss://ws.sepolia-api.lisk.com](wss://ws.sepolia-api.lisk.com)              |
+| **Chain ID** | 4202                                                                        |
+| **Currency Symbol** | ETH                                                                         |
+| **Block Explorer** | https://sepolia-blockscout.lisk.com/                                        |
```
Comment (matjazv) [inline docs/about-lisk/network-info.md]:
```suggestion
| **Block Explorer** | https://sepolia-blockscout.lisk.com                                        |
```

File: docs/building-on-lisk/contracts.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+description: ''
+
+---
+
+# Contracts
+A reference page containing information about all the contracts deployed by Lisk on different networks.
+
+## Lisk Sepolia Testnet
+
+The following contracts have been deployed by Lisk on the Lisk Sepolia Testnet.
```
Comment (matjazv) [inline docs/building-on-lisk/contracts.md]: Not all contracts were deployed on `Lisk Sepolia Testnet`. L1 contracts were deployed on `Ethereum Sepolia` network.

File: docs/building-on-lisk/contracts.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+id: contracts
+title: Contracts
+sidebar_position: 1
+slug: /contracts
+description: ''
+
+---
+
+# Contracts
+A reference page containing information about all the contracts deployed by Lisk on different networks.
```
Comment (matjazv) [inline docs/building-on-lisk/contracts.md]: `deployed by Lisk` is somehow not correct.
Contracts were deployed by Gelato, our RaaS provider.

`different networks` => Ethereum Sepolia and Lisk Sepolia test networks

File: docs/building-on-lisk/add-token-to-lisk.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+To add your token to the Lisk Token list, perform the following steps.
+
+### Step 1: Deploy your token on Lisk
+
+Use the recommended framework provided by Lisk's [Standard Bridge](network-info#lisk-sepolia-testnet) to deploy an ERC-20 contract for your token on Lisk.
+You can deploy contracts, and your token(s) using the [OptimismMintableERC20Factory](contracts).
+
+### Step 2: Submit details of your token
+
+Follow the instructions in the [ethereum-optimism.github.io repository's README](https://github.com/ethereum-optimism/ethereum-optimism.github.io?tab=readme-ov-file#superchain-token-list) and submit a pull request containing the required details for your token.
```
Comment (matjazv) [inline docs/building-on-lisk/add-token-to-lisk.md:60]: As far as I know, we'll need to have our own repository for tokens which will be listed inside [Lisk Standard Bridge](https://sepolia-bridge.lisk.com/bridge/lisk-sepolia-testnet).

Inside Optimism repository, PRs opened there are used to list tokens here: https://app.optimism.io/bridge/deposit
I think each chain (Optimism, Base, Lisk) needs to have their own list of tokens...

**Review by matjazv — CHANGES_REQUESTED** [review]

File: docs/building-on-lisk/contracts.md (hunk 3 above)
Comment (TalhaMaliktz) [reply docs/building-on-lisk/contracts.md]: Can you please name the contracts deployed on L1 and L2? I can add a column to differentiate between the contracts deployed on L1 and L2!

Comment (matjazv) [reply docs/building-on-lisk/contracts.md]: I think only those who have prefix `L1` were deployed on L1 network.

File: docs/building-on-lisk/add-token-to-lisk.md (hunk 4 above)
Comment (TalhaMaliktz) [reply docs/building-on-lisk/add-token-to-lisk.md:60]: From what I could understand [Base uses the same repo](https://github.com/ethereum-optimism/ethereum-optimism.github.io?tab=readme-ov-file#base-tokens) to upload the list of Tokens deployed to its network and so do [other chains based on OP Stack](https://github.com/ethereum-optimism/ethereum-optimism.github.io/pull/617/files). Maybe you can go through the [README](https://github.com/ethereum-optimism/ethereum-optimism.github.io/pull/617/files) and help me in clarifying this?

File: docs/building-on-lisk/contracts.md (hunk 3 above)
Comment (TalhaMaliktz) [reply docs/building-on-lisk/contracts.md]: I have added a column and have re-arranged contracts accordingly.

File: docs/about-lisk/network-info.md [hunk 5]
```diff
@@ -25,14 +25,14 @@ The Lisk L2 is available on the following network:
...
-| Lisk Testnet Bridge | https://sepolia-bridge.lisk.com                                             |
-| Lisk Testnet Faucet | https://app.optimism.io/faucet                                              |
+| **Network Name** | Lisk Sepolia Testnet                                                        |
+| **HTTP RPC Endpoint** | [https://rpc.sepolia-api.lisk.com](https://rpc.sepolia-api.lisk.com)        |
+| **WS RPC Endpoint** | [wss://ws.sepolia-api.lisk.com](wss://ws.sepolia-api.lisk.com)              |
+| **Chain ID** | 4202                                                                        |
+| **Currency Symbol** | ETH                                                                         |
+| **Block Explorer** | https://sepolia-blockscout.lisk.com                                        |
+| **Lisk Testnet Bridge** | https://sepolia-bridge.lisk.com                                             |
+| **Lisk Testnet Faucet** | https://app.optimism.io/faucet                                              |
```
Comment (Tschakki) [inline docs/about-lisk/network-info.md]: please adjust whitespace of the table, so columns are clear

File: docs/building-on-lisk/contracts.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+| **ProxyAdmin**                    | 0x5Db9F05921d8d5a6a157F6f49c411cc0e46c6330 | L2 |
+| **SystemConfig**                   | 0xF54791059df4a12BA461b881B4080Ae81a1d0AC0 | L2 |
+| **AddressManager**                | 0x27Bb4A7cd8FB20cb816BF4Aac668BF841bb3D5d3 | L2 |
+| **OptimismPortal**                | 0xe3d90F21490686Ec7eF37BE788E02dfC12787264 | L2 |
+| **ProtocolVersions**              | 0x1Bb17EB31bDdFd30F63D4FAe6c8Eb85D9e9b1f48 | L2 |
+| **OptimismMintableERC20Factory**  | 0x269d632C1E518a922C30C749cFD3f82Eb5C779B0 | L2 |
+
+
+:::tip
+For more information about the purpose of each smart contract mentioned above, see [**OP Mainnet > Smart Contracts**](https://l2beat.com/scaling/projects/optimism#contracts) section.
```
Comment (Tschakki) [inline docs/building-on-lisk/contracts.md]: This doesn't include descriptions for all contracts, and it is quite cumbersome to find the descriptions in general. Imho it wouldn't hurt to include a short description (max 1-2 sentences) in the table.

File: docs/building-on-lisk/contracts.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+# Contracts
+A reference page containing information about all the contracts deployed on different networks.
+
+## Sepolia Testnet (Ethereum & Lisk) 
+
+The following L1 and L2 contracts have been deployed to their respective network such as the **Ethereum Sepolia Testnet (L1)** and **Lisk Sepolia Testnet (L2)**.
+
+| Name                              | Value                                      | Deployed on |
+| :--------------                   | :----------------------------------------  |:----------  |
+| **L1ERC721Bridge**                | 0xb4E988CF1aD8C361D56118437502A8f11C7FaA01 | L1 |
```
Comment (Tschakki) [inline docs/building-on-lisk/contracts.md]: Can you please add explorer links to the respective contracts?

File: docs/building-on-lisk/add-token-to-lisk.md [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+---
+
+
+# The Lisk Token List
+It is possible to submit an ERC-20 contract for bridging with Lisk which has already been deployed on Ethereum.
+Lisk uses [Optimism's Superchain token list](https://github.com/ethereum-optimism/ethereum-optimism.github.io/blob/master/optimism.tokenlist.json) as a reference for tokens that have been deployed on Lisk.
+
+:::warning
+
+Tokens approved in the GitHub repository are not necessarily listed on the Lisk Bridge.
```
Comment (Tschakki) [inline docs/building-on-lisk/add-token-to-lisk.md]:
```suggestion
Tokens approved in the GitHub repository are not necessarily listed on the [Lisk Bridge](https://sepolia-bridge.lisk.com).
```

File: docs/building-on-lisk/add-token-to-lisk.md [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+:::
+---
+
+## Adding your token to the list
+
+To add your token to the Lisk Token list, perform the following steps.
+
+### Step 1: Deploy your token on Lisk
+
+Use the recommended framework provided by Lisk's [Standard Bridge](network-info#lisk-sepolia-testnet) to deploy an ERC-20 contract for your token on Lisk.
```
Comment (Tschakki) [inline docs/building-on-lisk/add-token-to-lisk.md]: Why not link directly to the bridge?
```suggestion
Use the recommended framework provided by Lisk's [Standard Bridge](https://sepolia-bridge.lisk.com/) to deploy an ERC-20 contract for your token on Lisk.
```

Also, I don't understand this step. How can I use the bridge to deploy ERC-20? Afaik, we currently only support ETH on the bridge. What is meant with "recommended framework"?

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: docs/building-on-lisk/add-token-to-lisk.md (hunk 4 above)
Comment (matjazv) [reply docs/building-on-lisk/add-token-to-lisk.md:60]: I'm not sure how this will work for Lisk Bridge.

This is actually a question for Gelato, which procedure they will use to list tokens inside Lisk Bridge.
They can use list inside Optimism repository but this doesn't strictly need to be true.

File: docs/building-on-lisk/add-token-to-lisk.md (hunk 9 above)
Comment (TalhaMaliktz) [reply docs/building-on-lisk/add-token-to-lisk.md]:
```suggestion
Use the recommended framework provided by Lisk's [Standard Bridge](https://sepolia-bridge.lisk.com) to deploy an ERC-20 contract for your token on Lisk.
```

**Comment by TalhaMaliktz** [comment]
@Tschakki and @matjazv,  I have incorporated your feedback on the **contracts page** and have separated it from this PR. You can take a look at the updated version here: https://github.com/LiskHQ/lisk-documentation/pull/49

File: docs/building-on-lisk/contracts.md (hunk 6 above)
Comment (TalhaMaliktz) [reply docs/building-on-lisk/contracts.md]: Feedback is applied in the PR mentioned below!

File: docs/building-on-lisk/contracts.md (hunk 7 above)
Comment (TalhaMaliktz) [reply docs/building-on-lisk/contracts.md]: Feedback is applied in the PR mentioned below!

File: docs/building-on-lisk/add-token-to-lisk.md (hunk 4 above)
Comment (TalhaMaliktz) [reply docs/building-on-lisk/add-token-to-lisk.md:60]: For now, this process is TBD so, according to Shu, we will only be able to roll out information about deploying an ERC-20 Contract on the Lisk network. I have aligned the content accordingly and have asked for your review.

File: docs/building-on-lisk/add-token-to-lisk.md [hunk 10]
```diff
@@ -0,0 +1,57 @@
...
+
+<!-- ### Step 2: Submit details of your token
+
+Follow the instructions in the [ethereum-optimism.github.io repository's README](https://github.com/ethereum-optimism/ethereum-optimism.github.io?tab=readme-ov-file#superchain-token-list) and submit a pull request containing the required details for your token.
+You must specify a section for `lisk-sepolia` and/or `lisk` in your token's `data.json` file.
+For more information, check out the currently active [pull requests](https://github.com/ethereum-optimism/ethereum-optimism.github.io/pulls) for adding an ERC-20 token to the Lisk network.
+
+### Step 3: Await final approval
+
+Tokens approved in the GitHub repository are not necessarily listed on the Lisk Bridge and are not guaranteed or automatic.
```
Comment (matjazv) [inline docs/building-on-lisk/add-token-to-lisk.md]:
```suggestion
Tokens approved in the GitHub repository are not necessarily listed on the Lisk Bridge; their listing is neither guaranteed nor automatic.
```

File: docs/building-on-lisk/add-token-to-lisk.md [hunk 11]
```diff
@@ -0,0 +1,57 @@
...
+::: -->
+
+
+<!-- ## Adding your token to the list
+
+To add your token to the Lisk Token list, perform the following steps. -->
+
+### Deploying your token on Lisk
+
+Use the recommended framework provided by Lisk's [Standard Bridge](https://sepolia-bridge.lisk.com) to deploy an ERC-20 contract for your token on Lisk.
```
Comment (matjazv) [inline docs/building-on-lisk/add-token-to-lisk.md]: I don't understand meaning of this statement.
Could you rephrase it?

**Review by matjazv — CHANGES_REQUESTED** [review]

File: docs/building-on-lisk/add-token-to-lisk.md (hunk 11 above)
Comment (TalhaMaliktz) [reply docs/building-on-lisk/add-token-to-lisk.md]: Done

File: docs/building-on-lisk/add-token-to-lisk.md [hunk 12]
```diff
@@ -0,0 +1,57 @@
...
+    'Ethereum',
+    'Lisk Mainnet',
+    'Optimism Superchain token list',
+  ]
+toc_max_heading_level: 4
+---
+
+
+# Adding ERC-20 Token to Lisk
+It is possible to deploy an ERC-20 contract for bridging with Lisk that has already been deployed on Ethereum.
```
Comment (Tschakki) [inline docs/building-on-lisk/add-token-to-lisk.md]:
```suggestion
This tutorial is meant for developers who already have an existing ERC-20 token on Ethereum and want to create a bridged representation of that token on Lisk. In this guide you'll learn how to deploy a custom ERC-20 token from Ethereum to Lisk, using the Standard Bridge system. 
```

Please give more context in the introduction

File: docs/building-on-lisk/add-token-to-lisk.md [hunk 13]
```diff
@@ -0,0 +1,57 @@
...
+::: -->
+
+
+<!-- ## Adding your token to the list
+
+To add your token to the Lisk Token list, perform the following steps. -->
+
+### Deploying your token on Lisk
+
+To deploy a ERC-20 token by using the [OptimismMintableERC20Factory](https://github.com/ethereum-optimism/specs/blob/main/specs/protocol/predeploys.md#optimismmintableerc20factory), follow the steps mentioned in the [Create an L2 ERC-20 Token](https://docs.optimism.io/builders/dapp-developers/tutorials/standard-bridge-custom-token#create-an-l2-erc-20-token) section.
```
Comment (Tschakki) [inline docs/building-on-lisk/add-token-to-lisk.md]: Afaik, `OptimismMintableERC20Factory` is used for deploying standard tokens, and `IOptimismMintableERC20` is used for custom tokens. Please do not mix that. Also, if both is already possible for Lisk (standard & custom token), then explain both options equally. Otherwise, only explain one option.

File: docs/building-on-lisk/add-token-to-lisk.md [hunk 14]
```diff
@@ -0,0 +1,57 @@
...
+
+Furthermore, to learn more about bridging various types of ERC-20 tokens to the Lisk Network, see the following guides:
+
+- [Bridging ERC-20 Tokens to OP Mainnet With the Optimism SDK](https://docs.optimism.io/builders/dapp-developers/tutorials/cross-dom-bridge-erc20)
```
Comment (Tschakki) [inline docs/building-on-lisk/add-token-to-lisk.md]:
```suggestion
```

Would remove this

File: docs/building-on-lisk/add-token-to-lisk.md [hunk 15]
```diff
@@ -0,0 +1,57 @@
...
+- [Bridging Your Standard ERC-20 Token to OP Mainnet Using the Standard Bridge](https://docs.optimism.io/builders/dapp-developers/tutorials/standard-bridge-standard-token)
```
Comment (Tschakki) [inline docs/building-on-lisk/add-token-to-lisk.md]: Currently I am afraid we should include the main steps in our docs, because we need to update `TUTORIAL_RPC_URL` for the users. 

But imho this is a perfect example, where it would be awesome it OP docs would include `TUTORIAL_RPC_URL` for Lisk as well, e.g. in form of a tab. We should ask them if they can include that in the OP docs, then we dont need to copy the steps.

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: docs/building-on-lisk/add-token-to-lisk.md (hunk 13 above)
Comment (matjazv) [reply docs/building-on-lisk/add-token-to-lisk.md]: `IOptimismMintableERC20` is interface which every ERC-20 token needs to implement if that token wants to be bridged token. So to deploy ERC-20 token to Lisk Network, it needs to implement `IOptimismMintableERC20` interface.

`OptimismMintableERC20Factory` is a smart contract which may be used to deploy ERC-20 tokens to LIsk Network.
It has two functions `createStandardL2Token(address _remoteToken, string memory _name, string memory symbol)` and `createOptimismMintableERC20(address _remoteToken, string memory _name, string memory _symbol)` which can be called to "automatically" create and deploy ERC-20 token on LIsk Network.
(second function deploys mintable ERC-20 token and first one not).

File: docs/building-on-lisk/add-token-to-lisk.md (hunk 12 above)
Comment (TalhaMaliktz) [reply docs/building-on-lisk/add-token-to-lisk.md]:
```suggestion
This tutorial is meant for developers with an existing ERC-20 token on Ethereum who want to deploy the same token on Lisk.
In this guide, you'll learn how to deploy a custom ERC-20 token from Ethereum to Lisk.
```

File: docs/building-on-lisk/add-token-to-lisk.md (hunk 14 above)
Comment (TalhaMaliktz) [reply docs/building-on-lisk/add-token-to-lisk.md]: Why 🤔 ?

File: docs/building-on-lisk/add-token-to-lisk.md (hunk 15 above)
Comment (TalhaMaliktz) [reply docs/building-on-lisk/add-token-to-lisk.md]: We can mention that whilst following the mentioned guide, update the `TUTORIAL_RPC_URL` to `https://rpc.sepolia-api.lisk.com/` the rest of the steps are the same for everyone. What do you say?

Comment (Tschakki) [reply docs/building-on-lisk/add-token-to-lisk.md]: yes, that would be an option

File: docs/building-on-lisk/add-token-to-lisk.md (hunk 13 above)
Comment (Tschakki) [reply docs/building-on-lisk/add-token-to-lisk.md]: @TalhaMaliktz Just to clarify, I am referring to the fact that you mention `OptimismMintableERC20Factory` and then link to the guide explaining how to bridge custom tokens, although this is the contract used for bridging standard tokens. Please adjust the content accordingly

File: docs/building-on-lisk/add-token-to-lisk.md (hunk 14 above)
Comment (Tschakki) [reply docs/building-on-lisk/add-token-to-lisk.md]: This guide is for bridging existing tokens, it is not relevant in this context.

File: docs/building-on-lisk/add-token-to-lisk.md (hunk 13 above)
Comment (TalhaMaliktz) [reply docs/building-on-lisk/add-token-to-lisk.md]: @matjazv and @Tschakki, I have re-aligned the text based on the discussed feedback and now it has details for deploying a standard and a custom ERC-20 token. Please take another look.

File: docs/building-on-lisk/add-token-to-lisk.md [hunk 16]
```diff
@@ -0,0 +1,68 @@
...
+
+:::important
+While following the aforementioned guide, update the `TUTORIAL_RPC_URL` variable to `https://rpc.sepolia-api.lisk.com/` to connect with Lisk Sepolia network.
+The rest of the steps are the same for Lisk as they are for Optimism.
+:::
+
+
+### Deploying a Custom ERC-20 token
+
+To deploy a Custom ERC-20 token, one needs to implement all the functions defined by the [IOptimismMintableERC20](https://github.com/ethereum-optimism/optimism/blob/v1.1.4/packages/contracts-bedrock/src/universal/IOptimismMintableERC20.sol) interface such as the logic for the `mint`, `burn`, and `l1Token`, functions of the `IOptimismMintableERC20` interface.
```
Comment (matjazv) [inline docs/building-on-lisk/add-token-to-lisk.md]: `mint`, `burn`, `remoteToken` and `bridge`

**Review by matjazv — APPROVED** [review]
One small change suggested.

File: docs/building-on-lisk/add-token-to-lisk.md (hunk 16 above)
Comment (TalhaMaliktz) [inline docs/building-on-lisk/add-token-to-lisk.md]:
```suggestion
To deploy a Custom ERC-20 token, one needs to implement all the functions defined by the [IOptimismMintableERC20](https://github.com/ethereum-optimism/optimism/blob/v1.1.4/packages/contracts-bedrock/src/universal/IOptimismMintableERC20.sol) interface such as the logic for the `mint`, `burn`, `remoteToken` and `bridge`, functions of the `IOptimismMintableERC20` interface.
```

File: docs/building-on-lisk/add-token-to-lisk.md [hunk 17]
```diff
@@ -0,0 +1,68 @@
...
+:::important
+While following the aforementioned guide, update the `TUTORIAL_RPC_URL` variable to `https://rpc.sepolia-api.lisk.com/` to connect with Lisk Sepolia network.
+The rest of the steps are the same for Lisk as they are for Optimism.
+:::
+
+
+### Deploying a Custom ERC-20 token
+
+To deploy a Custom ERC-20 token, one needs to implement all the functions defined by the [IOptimismMintableERC20](https://github.com/ethereum-optimism/optimism/blob/v1.1.4/packages/contracts-bedrock/src/universal/IOptimismMintableERC20.sol) interface such as the logic for the `mint`, `burn`, `remoteToken` and `bridge`, functions of the `IOptimismMintableERC20` interface.
+To do the aforementioned, follow the steps mentioned in the [Bridging Your Custom ERC-20 Token to OP Mainnet Using the Standard Bridge](https://docs.optimism.io/builders/dapp-developers/tutorials/standard-bridge-custom-token) guide.
```
Comment (Tschakki) [inline docs/building-on-lisk/add-token-to-lisk.md]:
```suggestion
To do the aforementioned, follow the steps mentioned in the [Bridging Your Custom ERC-20 Token to OP Mainnet Using the Standard Bridge](https://docs.optimism.io/builders/dapp-developers/tutorials/standard-bridge-custom-token#create-an-l2-erc-20-token) guide.
```

File: docs/building-on-lisk/add-token-to-lisk.md [hunk 18]
```diff
@@ -0,0 +1,68 @@
...
+
+
```
Comment (Tschakki) [inline docs/building-on-lisk/add-token-to-lisk.md:52]:
```suggestion
:::important
While following the aforementioned guide, ensure your wallet is connected to Lisk Sepolia in step 6.
The rest of the steps are the same for Lisk as they are for Optimism.
:::
```

File: docs/building-on-lisk/add-token-to-lisk.md [hunk 19]
```diff
@@ -0,0 +1,68 @@
+---
+title: Adding ERC-20 Token to Lisk
```
Comment (Tschakki) [inline docs/building-on-lisk/add-token-to-lisk.md]:
```suggestion
title: How to add your ERC-20 Token to Lisk
```

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: docs/building-on-lisk/add-token-to-lisk.md (hunk 19 above)
Comment (TalhaMaliktz) [reply docs/building-on-lisk/add-token-to-lisk.md]: The existing title suits more and is in sync with the titles of other pages so I would like to keep it as it is.

Comment (Tschakki) [reply docs/building-on-lisk/add-token-to-lisk.md]: hmm imho it is ok to use "How to..." sometimes, and here it sounds better to me personally.

But anyway i don't have a strong opinion. If we go for your version, I think it should be updated to

```suggestion
title: Adding ERC-20 Tokens to Lisk
```

or

```suggestion
title: Adding an ERC-20 Token to Lisk
```

File: docs/building-on-lisk/add-token-to-lisk.md [hunk 20]
```diff
@@ -0,0 +1,71 @@
...
+    'Sepolia',
+    'Ethereum',
+    'Lisk Mainnet',
+    'Optimism Superchain token list',
+  ]
+toc_max_heading_level: 4
+---
+
+
+# Adding ERC-20 Token to Lisk
```
Comment (Tschakki) [inline docs/building-on-lisk/add-token-to-lisk.md]:
```suggestion
# Adding an ERC-20 Token to Lisk
```

**Review by Tschakki — APPROVED** [review]
LGTM 🎉
